### ISSUE TITLE
Powering up the T-HMI board via the 5V DC connector does not work

### ISSUE THREAD

**Issue by xcyb0rg** [Xinyuan-LilyGO/T-HMI#19]
Hi,

As long as I power the board via the USB-C connector everything works fine. I've integrated T-HMI with Home Assistant via ESPHome and I'm happy with the overall results.

However, when I'm trying to power up the board via the 5V DC connector, the board doesn't seem to boot up. The little red LED also does not light up. I measured the voltage on the board, it definitely gets power.

I was in a similar position with a T-Display S3 board and realized that I need to set the GPIO15 HIGH on boot for the board to work with the DC connector. On the T-HMI I've tried setting GPIO10 (PWR_EN) and GPIO14 (PWR_ON) to HIGH on boot, but unfortunately the board does not power up.

I can't debug things further, because I would need to plug in the USB-C connector. But, if I do that, the board will get power via USB...

Is there any other GPIO pin that needs to be set in a certain way for the power to work via the DC connector?

Thank you!

**Comment by xcyb0rg** [comment]
Thank you, indeed, bringing a magnet close to the Reed Switch changed the behavior to some degree. As long as I have the magnet close the board connects to WiFi and is visible on the network. But for some reason I cannot access it on the network. The board is running esphome, but without access to it either via USB or TCP I cannot debug things further.

I eventually just powered the board via USB for my project and abandoned the idea of using the separate DC input.

**Comment by teastainGit** [comment]
Mihai xcyb0rg, The Magnet +  Reed switch is the craziest thing!
I can run on battery by observing these rules:
```
  pinMode(PWR_EN_PIN, OUTPUT);
  digitalWrite(PWR_EN_PIN, HIGH);

  pinMode(PWR_ON_PIN, OUTPUT);
  digitalWrite(PWR_ON_PIN, HIGH);
  //Battery operation works OK, but you need to press the
  //right hand button to boot off of battery
```

No magnet needed!
Please try again!
-Terry

**Comment by xcyb0rg** [comment]
I don't want to comment on the Reed Switch part, I can't imagine what the purpose of the switch is on the power input. And I trust you that somehow things can work by setting PWR_EN and PWR_ON as high. But I'm using ESPHome, where things are a little bit limited. 

I tried writing a very small firmware with Arduino and I've seen things tend to work. But when I'm using ESPHome, even though theoretically I set the same pins as high, it doesn't power on. 
Without USB plugged in I can't debug anything. And since wireless doesn't come up either, then... I'm stuck.

The workaround was to power the board via USB :/
